inject-java-test/src/test/groovy/io/micronaut/inject/annotation/AnnotationTransformerSpec.groovy

```diff
@@ -12,7 +12,8 @@ package test;
 @io.micronaut.inject.annotation.ToTransform
 @javax.inject.Singleton
 class Test {
-
+    void test(@io.micronaut.core.annotation.Nullable String test) {
+    }
```

Comment:
How is this change relevant for this test?

Reply:
Don't think it is, will cleanup it up